app/schemas/__init__.py

```diff
@@ -1,3 +1,8 @@
+from .common_names import (
```

Comment:
Okay, so all the schemas that we have will import/inherit those? right?

Reply:
Correct.

Reply:
It seems like we would still need to make our own Schema because the classes defined there have a lot of extra info that we haven't put in the database. In that case what do you suggest I should do?

Reply:
As for extra attributes, we can add them to the database. If there are cases where we need a different structure, we can still create new base models. In the end, it's not a strict thing and the idea is to reuse stuff that exists when we can, but it doesn't mean we should force ourselves to do this, especially if it adds complexity.